.github/workflows/generate-kubelet-flags.yaml

```diff
@@ -0,0 +1,48 @@
+name: Generate valid Kubelet Flags per k8s version
+on:
+  workflow_dispatch: {}
+
+jobs:
+  generate-kubelet-flags:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up containerd
+      uses: crazy-max/ghaction-setup-containerd@v2
+    - uses: actions/checkout@v3
+    - run: |
+        set -x
+
+        retrycmd_get_tarball() {
+          tar_retries=$1; wait_sleep=$2; tarball=$3; url=$4
+          echo "${tar_retries} retries"
+          for i in $(seq 1 $tar_retries); do
+              tar -tzf $tarball && break || \
+              if [ $i -eq $tar_retries ]; then
+                  return 1
+              else
+                  timeout 60 curl -fsSL $url -o $tarball
+                  sleep $wait_sleep
+              fi
+          done
+        }
+        
+        sudo swapoff -a
+        sudo ctr version
+        sudo systemctl status containerd
+        
+        sed '$ d' parts/linux/cloud-init/artifacts/manifest.json > temporary_manifest.json
+        KUBE_BINARY_VERSIONS="$(jq -r .kubernetes.versions[] temporary_manifest.json)"
+        for KUBE_BINARY_VERSION in $KUBE_BINARY_VERSIONS; do
```

Comment:
can we move this whole thing to a script and invoke the script for readability?

syntax highlighting and spacing in embedded yaml is rough

Reply:
done